RecoCTPPS/TotemRPLocal/plugins/CTPPSLocalTrackLiteProducer.cc

```diff
@@ -172,8 +170,10 @@ CTPPSLocalTrackLiteProducer::fillDescriptions( edm::ConfigurationDescriptions& d
   desc.add<bool>( "doNothing", true ) // disable the module by default
     ->setComment( "disable the module" );
 
-  desc.add<std::vector<double> >("pixelTrackTxRange",std::vector<double>({-0.03,0.03}) );
-  desc.add<std::vector<double> >("pixelTrackTyRange",std::vector<double>({-0.04,0.04}) );
+  desc.add<double>("pixelTrackTxMin",-10.0);
+  desc.add<double>("pixelTrackTxMax", 10.0);
```

Comment:
is this increase in range intentional?
what are the units, BTW?
a comment here would be nice to improve self-documentation

Reply:
The range increase is intentional as it was required by the analysis group to remove any kind of cuts when propagating pixel tracks to trackLites. The units are the same of tx and ty as defined in [https://github.com/cms-sw/cmssw/blob/0a4b35a9c1ca2d78952b75d9b95887f7a82f5a34/DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h#L179](url), which means that they are both pure numbers. 
Should I specify this here as well?